lib/newspaper_works/data.rb

```diff
@@ -8,5 +8,24 @@ module NewspaperWorks
   # Module for data access helper / adapter classes supporting, enhancing
   #   NewspaperWorks work models
   module Data
+    # Handler for after_create_fileset, to be called by block subscribing to
+    #   and overriding default Hyrax `:after_create_fileset` handler, via
+    #   app integrating newspaper_works.
+    def self.handle_after_create_fileset(file_set, user)
+      handle_queued_derivative_attachments(file_set)
+      # Hyrax queues this job by default, and since newspaper_works
+      #   overrides the single subscriber Hyrax uses to do so, we
+      #   must call this here:
+      FileSetAttachedEventJob.perform_later(file_set, user)
+    end
+
+    def self.handle_queued_derivative_attachments(file_set)
```

Comment:
I might be misunderstanding here. Does this need to be a class method? Doesn't appear to be called anywhere else but within this module.

Reply:
I did not want to create a class just for these two methods in this module.  The `self.` spelling was necessary to make the methods module-scoped.  This method is called by the app, via the generated config [here](https://github.com/marriott-library/newspaper_works/pull/83/files#diff-4ac104eb61d292e014b94b96f6cc9377R25).